catalog/global-pasture-watch/projects_global-pasture-watch_assets_gsvh-30m_v1_short-veg-height_m

```diff
@@ -35,11 +35,12 @@ local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
     ee.host_provider(self_ee_catalog_url),
   ],
   extent: ee.extent_global('2000-01-01T00:00:00Z', null),
+  'gee:units': units.meter,
```

Comment:
this needs to be in the band section:

'eo:bands': [
      {
        name: 'height',
        description: 'Median vegetation height',
        gsd: 30,
        'gee:scale': 0.1,
        'gee:units': units.meter,
      }